### PR TITLE
very very long text

### PR DESCRIPTION
**Pull request by stubbEnable**

### CODE REVIEW

File: src/main/java/mindustry/plugin/ioMain.java [hunk 1]
```diff
@@ -130,7 +130,7 @@ public ioMain() {
                 }
 
                 if(bannedNames.contains(player.name))
-                    player.con.kick("Influx Capacitor failed.");
```
Comment (fuzzbuck) [inline src/main/java/mindustry/plugin/ioMain.java:133]: keep this

File: src/main/java/mindustry/plugin/commands/ModeratorCommands.java [hunk 2]
```diff
@@ -75,15 +75,19 @@ public void registerCommands(CommandHandler handler){
...
                 PlayerData pd = Database.getData(uuid);
                 if(pd != null){
+                    if (Integer.parseInt(args[1]) > 999999){
+                        ctx.sendEmbed(false, "<:im_ok:772010005682716682> There was an error trying to execute that command");
+                        return;
+                    }
                     long until = Instant.now().getEpochSecond() + Integer.parseInt(args[1]) * 60;
                     pd.bannedUntil = until;
-                    pd.banReason = (args.length >= 3 ? args[2] : "not specified") + "\n" + "[accent]Until: " + epochToString(until) + "\n[accent]Ban ID:[] " + player.uuid().substring(0, 4);
+                    pd.banReason = (args.length >= 3 ? args[2] : "not specified") + "\n" + "[red]You are banned from this server.[][accent]\nReason:[][red] " + pd.banReason + "\n[accent]Until: " + epochToString(until) + "\n[accent]Ban ID:[] " + player.uuid().substring(0, 4) + "\n\nIf you think you were banned by mistake or wish to appeal your ban, make an appeal at:\n[cyan]https://discord.mindustry.io/[]";
```
Comment (fuzzbuck) [inline src/main/java/mindustry/plugin/commands/ModeratorCommands.java:84]: this does not make any sense

File: src/main/java/mindustry/plugin/commands/ModeratorCommands.java [hunk 3]
```diff
@@ -75,15 +75,19 @@ public void registerCommands(CommandHandler handler){
...
                     Database.updateData(uuid, pd);
 
                     HashMap<String, String> fields = new HashMap<>();
                     fields.put("UUID", uuid);
                     ctx.sendEmbed(true, ":hammer: the ban hammer has been swung at " + escapeCharacters(player.name), "reason: *" + escapeColorCodes(pd.banReason) + "*", fields, false);
-                    player.con.kick(KickReason.banned);
+                    player.con.kick("[red]You are banned from this server.[][accent]\nReason:[][red] " + pd.banReason + "\n[accent]Until: " + epochToString(until) + "\n[accent]Ban ID:[] " + player.uuid().substring(0, 4) + "\n\nIf you think you were banned by mistake or wish to appeal your ban, make an appeal at:\n[cyan]https://discord.mindustry.io/[]");
```
Comment (fuzzbuck) [inline src/main/java/mindustry/plugin/commands/ModeratorCommands.java:90]: please look above and realise that this does not make any sense

**Review by fuzzbuck — CHANGES_REQUESTED** [review]
bruh

File: src/main/java/mindustry/plugin/commands/ModeratorCommands.java (hunk 3 above)
Comment (fuzzbuck) [reply src/main/java/mindustry/plugin/commands/ModeratorCommands.java:90]: this should be player.con.kick(pd.banReason);

File: src/main/java/mindustry/plugin/utils/Funcs.java [hunk 4]
```diff
@@ -117,7 +117,7 @@ public static String escapeCharacters(String string){
     }
 
     public static String escapeColorCodes(String string){
-        return Strings.stripColors(string);
+        return Strings.stripColors(string).replaceAll("`", "");
```
Comment (fuzzbuck) [inline src/main/java/mindustry/plugin/utils/Funcs.java:120]: add this to escapeCharacters() instead of here

have some order you people

**Review by fuzzbuck — CHANGES_REQUESTED** [review]

File: src/main/java/mindustry/plugin/utils/Funcs.java (hunk 4 above)
Comment (stubbEnable) [reply src/main/java/mindustry/plugin/utils/Funcs.java]: not needed, removed it

**Pull request merged**